### PR TITLE
W-9025974 - Staff access review listview

### PR DESCRIPTION
**Pull request by seanpat09**
Screenshot of list view:

Screen Shot 2021-04-23 at 3 33 32 PM

# Critical Changes

# Changes
Adds a list view for the Review object called "Open Reviews".  Displays all reviews with that do not have a Status of "Completed"

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
- ~[ ] Any net new LWC work has JEST test coverage 50% or above~
- ~[ ] Default Sa11y tests pass for all LWC components~
- ~[ ] 🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary~
- ~[ ] 🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary~
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
- [x] Make sure that ACs are updated (if any gaps)
- [x] **All acceptance criteria have been met**
    - [x] Developer
    - [x] Code Reviewer
- [x] Pull Request contains draft release notes
- [ ] Labels, help text, and customer facing messages are reviewed by Docs
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by seanpat09** [comment]
[W-9025974 - Staff access review listview](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008aHzkIAE/view)

File: force-app/main/default/objects/Review__c/listViews/OpenReviews.listView-meta.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <columns>DueDate__c</columns>
+    <columns>Comments__c</columns>
+    <columns>AssignedTo__c</columns>
+    <filterScope>Everything</filterScope>
+    <filters>
+        <field>Status__c</field>
+        <operation>notEqual</operation>
+        <value>Completed</value>
+    </filters>
+    <label>Open Reviews</label>
```
Comment (seanpat09) [inline force-app/main/default/objects/Review__c/listViews/OpenReviews.listView-meta.xml:16]: @nicolersmith This is the name of the new list view

Screen Shot 2021-04-23 at 3 33 32 PM

**Review by spelak-salesforce — APPROVED** [review]
Looks great!

**Comment by nicolersmith** [comment]
@seanpat09 Does the list view need a description? Not sure if that's even a thing. :)

**Comment by seanpat09** [comment]
> @seanpat09 Does the list view need a description? Not sure if that's even a thing. :)

Haha just checked my org to confirm because I wasn't sure myself - list view do not have descriptions.

**Pull request merged**